config/core/manager_disable_kubeadm.yaml

```diff
@@ -0,0 +1,5 @@
+- op: replace
```

Comment:
I hit issues trying to just append/remove to the list of args. Replace seems to work fine, although I know it's not ideal.

Reply:
Can we rename this file to `manager_arguments.yaml` instead?